src/main/stackpack/resources/waitingfordata.md

```diff
@@ -1,75 +1,30 @@
-
-This mechanism allows you to connect using the client certificate and the private key. 
-The following new parameters are available :
-* `verify` - `True` or `False` depending on verifying the client certificate or not. By default, it's `True`.
-* `cert` - path containing the client side certificate
-* `keyfile` - path containing the private key for certificate
-
-_As an example, see the below config :_
-
-```
-# Section used for global SAP check config
-init_config: {}
-
-instances:
-    - host: TEST-01             # <sap_host_name> 
-      url: https://test-01      # <sap_host_url>   
-      user: test                # <username>
-      pass: test                # <password> 
-      verify: False
-      cert: /path/to/cert.pem   # <certificate_path>
-      keyfile: /path/to/key.pem # <keyfile_path>
-      domain: sap           # <optional>
-      environment: sap      # <optional>
-      tags:                 # <optional>
-        - customer:stackstate
-```
-**NOTE** - Make sure while using this mechanism, you have put `https` in the `url` of the config.
-
-Once the configuration changes are done as explained above, restart the StackState Agent(s) using the following command.
-
-```
-sudo /etc/init.d/stackstate-agent restart
-```
-
-Once the Agent is restarted, wait for the Agent to collect the data and send it to StackState.
+### The StackState SAP StackPack is waiting for data
+
+To begin collecting data from SAP, add the following configuration to StackState Agent V2:
+
+1. Edit the Agent integration configuration file `/etc/stackstate-agent/conf.d/sap.d/conf.yaml` to include details of your SAP instance:
+   - **host** - A unique name to identify the SAP instance.
+   - **url** - The URL to connect to the SAP instance. Use http for basic authentication (user/pass) and https for client certificate authentication.
+   - **user** and **pass** - SAP authentication details to use HTTP basic authentication. Requires an HTTP `url`. Use [secrets management](https://l.stackstate.com/ui-stackpack-secrets-management) to store passwords outside of the configuration file.
+   - **cert** and **keyfile** - SAP authentication details to use client certificate authentication. Requires an HTTPS `url`.
+
+   ```
+   # Section used for global SAP check config
+   init_config: {}
+   
+   instances:
+       - host: TEST-01             # <sap_host_name> 
+         url: https://test-01      # <sap_host_url>   
+         user: test                # <username>
+         pass: test                # <password>
+         cert: /path/to/cert.pem   # <certificate_path>
+         keyfile: /path/to/key.pem # <keyfile_path>
+   ```
+
+2. You can also add optional configuration, for details see the [SAP StackPack documentation \(docs.stackstate.com\)](https://l.stackstate.com/ui-sap-configure-check).
+3. [Restart the StackState Agent\(s\)](https://l.stackstate.com/ui-stackpack-restart-agent) to apply the configuration changes.
```

Comment:
Wouldn't it be nice to have the restart command here ?

Reply:
we agreed to always have a link out to the command and host it on the docs. this is true for all stackpacks